matchms/Spectrum.py

```diff
@@ -93,7 +85,13 @@ def clone(self):
         return clone
 
     def plot(self, intensity_from=0.0, intensity_to=None, with_histogram=False, with_expfit=False):
-        """An example docstring for a method."""
+        """To visually inspect a spectrum run ``spectrum.plot()``
+
+        .. figure:: _static/spectrum-plot-example.png
```

Comment:
Almost Spectrum html page is in api/ dir so 
```suggestion
        .. figure:: ../_static/spectrum-plot-example.png
```